dotnet/src/webdriver/Remote/W3CWireProtocolCommandInfoRepository.cs

```diff
@@ -143,6 +143,25 @@ protected override void InitializeCommandDictionary()
             this.TryAddCommand(DriverCommand.TouchFlick, new CommandInfo(CommandInfo.PostCommand, "/session/{sessionId}/touch/flick"));
 
             this.TryAddCommand(DriverCommand.UploadFile, new CommandInfo(CommandInfo.PostCommand, "/session/{sessionId}/file"));
+
+            // HTML5 commands
+            this.TryAddCommand(DriverCommand.GetLocation, new CommandInfo(CommandInfo.GetCommand, "/session/{sessionId}/location"));
```

Comment:
this file implies that the end points should be from the W3C?

These commands are not included there -
https://w3c.github.io/webdriver/webdriver-spec.html

Probably shouldn't have this here? But I'm not sure, since the 'Touch' API is included in this file, but that's not part of the spec either.

@jimevans will have to comment :)